install/kubernetes/helm/istio/values.yaml

```diff
@@ -210,7 +210,8 @@ gateways:
 
     loadBalancerIP: ""
     loadBalancerSourceRanges: {}
-    serviceAnnotations: {}
+    serviceAnnotations: 
```

Comment:
I am worried about the situation when customers deploy the istio-demo-auth.yaml as it is, in this case, this "serviceAnnotations" field will be empty, would this cause any issue?

Reply:
in istio-demo-auth.yaml sds is enabled by default, so ingress->sidecar won't work leave this empty, which is expected